k

```diff
@@ -5,11 +5,17 @@
 # KAIL_PATH - path to kail executable, defaults to "kail"
 # KUBESEAL_PATH - path to kubeseal executable, defaults to "kubeseal"
 # YQ_PATH - path to yq executable, defaults to "yq"
+# K_LISTEN_ON_ALL_INTERFACES - if set truthy, listen on 0.0.0.0 for local forwards
 
 require "date" # to protect against YAML bug present in certain psych versions: https://github.com/ruby/psych/pull/695
 require "yaml"
 require "tmpdir"
 require "uri"
+require "socket"
```

Comment:
I'm trying to be hardcore about avoiding `require` on the main path, to keep `k` execution as fast as possible. Maybe relying on `HOSTNAME` env var is good enough at this stage?